plugins/optimization-detective/hooks.php

```diff
@@ -12,6 +12,9 @@
 }
 // @codeCoverageIgnoreEnd
 
+// The addition of the following hooks is tested in Test_OD_Hooks::test_hooks_added() and Test_OD_Storage_Post_Type::test_add_hooks().
+
+// @codeCoverageIgnoreStart
 add_action( 'init', 'od_initialize_extensions', PHP_INT_MAX );
```

Comment:
Too bad, I guess there is no way to manually indicate that this is covered by the tests for the hooks added?

Reply:
I think the only way may be to have a test that removes all filters/actions, and then does a `require` on the `hooks.php` file. If after doing so all of the filters/actions have been added, then we know it worked. This would work for Optimization Detective, but it wouldn't work for other plugins that actually define functions in `hooks.php`. So maybe we should consider that later as part of #1789.

Note that the coverage of `uninstall.php` is handled similarly: https://github.com/WordPress/performance/blob/trunk/plugins/optimization-detective/tests/test-uninstall.php